sentry-samples/sentry-samples-spring-jakarta/build.gradle.kts

```diff
@@ -9,7 +8,7 @@ plugins {
     kotlin("jvm")
     alias(libs.plugins.kotlin.spring)
     id("war")
-    id(Config.BuildPlugins.gretty) version Config.BuildPlugins.grettyVersion
+    alias(libs.plugins.gretty)
```

Comment:
do we need to specify it in the root project with the version then?

Reply:
that’s a good point, i’ll do this.

it isn’t required but it helps maintain a single classpath for all projects. this build is a bit tricky since different subprojects use different versions of the same plugin (like spring) so the buildscript classpath cannot by synchronized 100% for now.

Reply:
Actually it looks like gretty depends on spring boot and since we don’t have spring boot in the root buildscript classpath, this causes conflicts.